api/entity/Genre.ts

```diff
@@ -1,7 +1,7 @@
-import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { BaseEntity, Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
 
 @Entity()
-export class Genre {
+export class Genre extends BaseEntity {
```

Comment:
Why do you extends the class with `BaseEntity `when you don't have a `constructor `?